tests/unit/test_urls.py

```diff
@@ -11,15 +11,15 @@
 def test_path_to_url_unix() -> None:
     assert path_to_url("/tmp/file") == "file:///tmp/file"
     path = os.path.join(os.getcwd(), "file")
-    assert path_to_url("file") == "file://" + path
+    assert path_to_url("file") == "file://" + urllib.parse.quote(path)
```

Comment:
Without this change, tests with fail if the working directory contains a non-urlsafe character.